src/kafka.rs

```diff
@@ -68,7 +68,7 @@ pub fn create_consumer() -> Result<StreamConsumer, KafkaError> {
         .set("auto.offset.reset", "beginning")
         .set("api.version.request", "false")
         .set("security.protocol", "plaintext")
-        .set("max.partition.fetch.bytes", "2097152")
```

Comment:
Denne verdien ble satt tidligere siden vi hadde flere meldinger som var for stor som feilet. Du kan prøve om det går men denne verdien ble økt med vilje.